src/pages/equipo.js

```diff
@@ -1,386 +1,413 @@
+                src={useBaseUrl("img/team/profile-thinner.jpg")}
+              />
+            )}
+          </Box>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <Box>
+            <h3 className="nameText">{name}</h3>
+            <p className="positionText">{position}</p>
+            <p className="bioText">{bio}</p>
+          </Box>
+          <Box className="socialMediaBox">
+            {linkedIn && (
+              <a href={linkedIn} target="_blank" className="socialMediaIcon">
+                <LinkedInIcon style={{ width: "30px", height: "30px" }} />
+              </a>
+            )}
+            {twitter && (
+              <a href={twitter} target="_blank" className="socialMediaIcon">
+                <TwitterIcon style={{ width: "30px", height: "30px" }} />
+              </a>
+            )}
+          </Box>
+        </Grid>
+      </Grid>
+    );
+  };
 
-    const TeamSection = () => {
-        return(
-            <Box className="containerSec">
-                <Box className={clsx("sectionNoPadding")}>
-                    <Box className={isDesktop? "doubleSpacingBox": ""}>
-                        <Box className="spacingBox">
-                            <h2>Equipo gerencial</h2>
-                        </Box>
-                        <Box>
-                            <Grid container>
-                                {ManagementTeam.map((person) => (
-                                    <Grid item xs={12} md={6} key={person.name}>
-                                        <Box className="teamPadding" style={{backgroundColor:isMobile? person.color: ""}}>
-                                            <PersonCard 
-                                                img={useBaseUrl(person.img)}
-                                                name={person.name}
-                                                position={person.position}
-                                                bio={person.bio}
-                                                linkedIn={person.linkedIn}
-                                                twitter={person.twitter}
-                                            />
-                                        </Box>
-                                    </Grid>
-                                ))}
-                            </Grid>
-                        </Box>
+  const TeamSection = () => {
+    return (
+      <Box className="containerSec">
+        <Box className={clsx("sectionNoPadding")}>
+          <Box className={isDesktop ? "doubleSpacingBox" : ""}>
+            <Box className="spacingBox">
+              <h2>Equipo gerencial</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {ManagementTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```

Comment:
Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop